zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -478,7 +534,43 @@ def deploy_client(self, req):
         return jsonobject.dumps(rsp)
 
     @replyerror
-    def get_facts(self, req):
+    def get_logical_pool_info(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.8.2)

`http` may be undefined, or defined from star imports

(F405)